### ISSUE TITLE
glossary: create stand-alone page

### ISSUE THREAD

**Issue by jorgeorpinel** [iterative/dvc.org#542]
It would be nice to have a **Glossary** menu item on the nav bar (or/and linked from tooltips) that opens a page where all these terms are listed (with `<a name='the-term'>` anchors. Extra steps:

- [ ] This is mainly to link from glossary descriptions (because we can't have tooltips inside tooltips)
  Note: currently **bold** words in tooltip texts mark other glossary entries;
- [ ] and to read the full list in case anyone is curious.

> Extracted from https://github.com/iterative/dvc.org/issues/518#issuecomment-519813441

- [ ] Check whether #550 is still necessary

**Comment by jorgeorpinel** [comment]
Maybe after #461 though.

**Comment by jorgeorpinel** [comment]
At this point I think we're definitely merging this idea and the basic concepts so lets close this in favor of #550?

**Comment by shcheklein** [comment]
Agreed!
